tests/unit/commands/server-test.js

```diff
@@ -126,4 +126,16 @@ describe('server command', function() {
       expect(ops.baseURL).to.equal('test', 'Uses the correct environment.');
     });
   });
+
+  it('host alias does not conflict with help alias', function() {
+    return new ServeCommand(options).validateAndRun([
+      '-ho', 'hostname'
```

Comment:
If that's the direction that aliases are going to go then I can absolutely do that.  I didn't notice any other commands with a capital letter alias and didn't want to break any unspoken convention.